### PR TITLE
(example) update github-pages example

### PR DESCRIPTION
**Pull request by vinaykulk621**
Removed `appDir: true` from experimental flag and code refactor.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e5742f2e44a1896438c1a0994662d6d84efccdaf

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e5742f2e44a1896438c1a0994662d6d84efccdaf

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e5742f2e44a1896438c1a0994662d6d84efccdaf

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by manovotny** [comment]
@vinaykulk621 thanks for the contribution! Any specific reason you're adding Inter to this example? If for no specific reason, let's remove it and get this merged.

**Comment by vinaykulk621** [comment]
When you use `create-next-app` it initializes the project with `Inter` out of the box. 
I was trying to keep the examples as close as possible to the `create-next-app` starter template.
Should i remove it?

**Comment by vinaykulk621** [comment]
@manovotny Any updates?

File: examples/github-pages/app/layout.tsx [hunk 1]
```diff
@@ -1,11 +1,20 @@
+import { Inter } from 'next/font/google'
```
Comment (styfle) [inline examples/github-pages/app/layout.tsx:1]:
```suggestion
import type { Metadata } from 'next'
import { Inter } from 'next/font/google'
```

File: examples/github-pages/app/layout.tsx [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+
+const inter = Inter({ subsets: ['latin'] })
+
+export const metadata = {
```
Comment (styfle) [inline examples/github-pages/app/layout.tsx:5]:
```suggestion
export const metadata: Metadata = {
```

File: examples/github-pages/app/layout.tsx [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
+  title: 'Next.js on github pages',
+  description: 'Deploy next.js web application on github pages',
```
Comment (styfle) [inline examples/github-pages/app/layout.tsx:7]:
```suggestion
  title: 'Next.js on GitHub Pages',
  description: 'A Next.js web application on GitHub Pages',
```

**Review by styfle — APPROVED** [review]

**Pull request merged**